### PR TITLE
Jaro Winkler + vdf.join

### PR DESCRIPTION
**Pull request by oualib**
- Formatting the code
 - Jaro + Jaro Winkler distances
 - vdf.join: improvement new operators

### CODE REVIEW

**Comment by oualib** [comment]
@tconsigny can you please look at this PR, it should be one of the last ones before v0.11.0 (and the last one which needs doc review)

File: verticapy/vdataframe.py [hunk 1]
```diff
@@ -7896,8 +7896,27 @@ def join(
...
+                     '<' : key1  < key2
+                     '>' : key1  > key2
+                    '<=' : key1 <= key2
+                    '>=' : key1 >= key2
+                 'llike' : key1 LIKE '%' || key2 || '%'
+                 'rlike' : key2 LIKE '%' || key1 || '%'
+           'linterpolate': key1 INTERPOLATE key2
+           'rinterpolate': key2 INTERPOLATE key1
+        Some operators need 5-tuples: (key1, key2, operator, operator2, x) where
+        operator2 is one simple operator (=, >, <, <=, >=), x a float or an integer
```
Comment (mail4umar) [inline verticapy/vdataframe.py]: x IS a float

File: verticapy/vdataframe.py [hunk 2]
```diff
@@ -7896,8 +7896,27 @@ def join(
     ----------
     input_relation: str/vDataFrame
         Relation to use to do the merging.
-    on: dict, optional
-        Dictionary of all different keys. The dict must be similar to the following:
+    on: dict / list, optional
+        List of 3-tuples. Each tuple must include (key1, key2, operator) where
```
Comment (mail4umar) [inline verticapy/vdataframe.py]: maybe write: If it is a list then....
and below we can write : if it a dict then

Maybe use two indents for each to make it more clear.

**Review by tconsigny — APPROVED** [review]

**Review by mail4umar — APPROVED** [review]

File: verticapy/vdataframe.py (hunk 1 above)
Comment (oualib) [reply verticapy/vdataframe.py]: for levenstein, it is an integer

File: verticapy/vdataframe.py (hunk 2 above)
Comment (oualib) [reply verticapy/vdataframe.py]: ok

**Pull request merged**